README.md

```diff
@@ -1,6 +1,16 @@
-# datafusion-tui (dft)
+# dft
 
-DataFusion-tui provides an extensible terminal based data analysis tool that uses [DataFusion](https://github.com/apache/arrow-datafusion) as query execution engines. It has drawn inspiration and several features from `datafusion-cli`. In contrast to `datafusion-cli` a focus of `dft` is to provide an interface for leveraging DataFusions extensibility (for example connecting to `ObjectStore`s or querying custom `TableProvider`s).
+`dft` provides two interfaces to the [DataFusion](https://github.com/apache/arrow-datafusion) query execution engine:
```

Comment:
I also took a shot at updating the description here to describe the two different UIs. Let me know what you think

Reply:
Looks good, and I like the scripting option.  Only comment from the updated description is that right now there isnt any MySQL integration.